modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptClosureAngularClientCodegen.java

```diff
@@ -64,15 +70,11 @@ public JavascriptClosureAngularClientCodegen() {
 
         typeMapping.put("binary", "string");
 
-        outputFolder = "generated-code/javascript-closure-angular";
-        modelTemplateFiles.put("model.mustache", ".js");
-        apiTemplateFiles.put("api.mustache", ".js");
-        embeddedTemplateDir = templateDir = "Javascript-Closure-Angular";
-        apiPackage = "API.Client";
-        modelPackage = "API.Client";
-
         cliOptions.add(new CliOption(CodegenConstants.HIDE_GENERATION_TIMESTAMP, "hides the timestamp when files were generated")
                 .defaultValue(Boolean.TRUE.toString()));
+        cliOptions.add(new CliOption(USE_ES6,
+                "use plain ES6 templates instead - drops Google Closure Compiler annotations")
```

Comment:
@miguelvaladez thanks for the enhancement. Would it be possible to keep the Google Closure Compiler annotations? I think that's what the users expect to get in the output when they use the `javascript-closure-angular` client generator.

Btw, we just merged https://github.com/swagger-api/swagger-codegen/pull/5593 to support ES6 in the `javascript` client generator.